src/services/selinux.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Created by Lucas Gracioso <contact@lbgracioso.net>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "cloysterhpc/selinux/selinux.h"
+#include "cloysterhpc/functions.h"
+#include "cloysterhpc/services/log.h"
+
+void SELinux::configurexCATPolicies()
+{
+    LOG_TRACE("Configuring xCAT SELinux policies")
+    configurexCATPolicyFile();
+    configurexCATgenimagePolicyFile();
+    configurexCATrsyncPolicyFile();
+}
+
+void SELinux::configurexCATPolicyFile()
+{
+    // Create SELinux policy file for xCAT
+    const std::string file = "/root/xCAT-httpd-read-tftpdir.te";
```

Comment:
Do we have a `/opt/cloysterhpc` directory right? We should use it instead of `/root`.